### PR TITLE
add WorkerService examples from HOME

### PR DESCRIPTION
**Pull request by TimothyMothra**
adding WorkerService examples from former HOME repo

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]

File: examples/WorkerServiceSDK/ConsoleAppWithApplicationInsights/ConsoleAppWithApplicationInsights/Program.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+            // Add or configure channel
+            services.AddSingleton<ITelemetryChannel>(new ServerTelemetryChannel() { StorageFolder = @"C:\temp\aisdkstorage" });
+
+            // Add custom TelemetryInitializer
+            services.AddSingleton<ITelemetryInitializer, MyCustomTelemetryInitializer>();
+
+            // Being a regular console app, there is no appsettings.json or configuration providers enabled by default.
+            // Hence instrumentation key must be specified here.
+            services.AddApplicationInsightsTelemetryWorkerService("0eb18de0-7268-473c-ae06-c87ca480d2d2");
```
Comment (avereshchak) [inline examples/WorkerServiceSDK/ConsoleAppWithApplicationInsights/ConsoleAppWithApplicationInsights/Program.cs:30]: Is this key expected here?

Comment (cijothomas) [reply examples/WorkerServiceSDK/ConsoleAppWithApplicationInsights/ConsoleAppWithApplicationInsights/Program.cs:30]: no. @TimothyMothra lets put "put_actual_ikey_here" thing here.

**Comment by asears** [comment]
Could these samples include some unit test project with moq/mocking?

**Pull request merged**